frontend/src/views/projectViews/ProjectDetailPage/ProjectDetailPage.tsx

```diff
@@ -66,42 +70,176 @@ export default function ProjectDetailPage() {
-                        <BsPersonFill />
-                    </NumberOfStudents>
-                </ClientContainer>
-
-                <CoachesContainer>
-                    {project.coaches.map((element, _index) => (
-                        <CoachContainer key={_index}>
-                            <CoachText>{element.name}</CoachText>
-                        </CoachContainer>
-                    ))}
-                </CoachesContainer>
-
-                <div>
-                    {students.map((element: StudentPlace, _index) => (
-                        <StudentPlaceholder studentPlace={element} key={_index} />
-                    ))}
-                </div>
-            </ProjectContainer>
-        </div>
+        <DragDropContext onDragEnd={result => onDragDrop(result)}>
+            <ProjectPageContainer>
+                <StudentListFilters />
+
+                <ProjectContainer>
+                    <GoBack onClick={() => navigate("/editions/" + editionId + "/projects/")}>
+                        <BiArrowBack />
+                        Overview
+                    </GoBack>
+
+                    <TitleAndEdit
+                        editing={editing}
+                        project={project}
+                        editedProject={editedProject}
+                        setEditedProject={setEditedProject}
+                        setEditing={setEditing}
+                        editProject={editProject}
+                        role={role!}
+                        handleShow={handleDeleteShow}
+                    />
+
+                    <ConfirmDelete
+                        visible={showDeleteModal}
+                        handleConfirm={handleDelete}
+                        handleClose={handleDeleteClose}
+                        name={project.name}
+                    ></ConfirmDelete>
+
+                    <ClientsContainer>
+                        <ProjectPartners
+                            project={project}
+                            editedProject={editedProject}
+                            setEditedProject={setEditedProject}
+                            editing={editing}
+                        />
+                        <NumberOfStudents>
+                            {studentAmount}
+                            <BsPersonFill />
+                        </NumberOfStudents>
+                    </ClientsContainer>
+
+                    <ProjectCoaches
+                        project={project}
+                        editedProject={editedProject}
+                        setEditedProject={setEditedProject}
+                        editing={editing}
+                    />
+
+                    <MoreInfoLink>
+                        <CreateButton
+                            showIcon={false}
+                            label="More info about this project"
+                            onClick={() => window.open(project.infoUrl)}
+                        />
+                    </MoreInfoLink>
```

Comment:
Seeing as the infoUrl is nullable, this should only be rendered if the url is not null. (Just use a basic `{project.infoUrl !== null && <Component />}` conditional render).
```suggestion
                    <MoreInfoLink>
                        <CreateButton
                            showIcon={false}
                            label="More info about this project"
                            onClick={() => window.open(project.infoUrl)}
                        />
                    </MoreInfoLink>
```